context/app/static/js/components/home-revision/AssayTypeBarChart/hooks.js

```diff
@@ -0,0 +1,13 @@
+import { useTheme } from '@material-ui/core/styles';
+
+function useChartPalette() {
+  const {
+    palette: { success, primary, secondary, error, warning, info },
+  } = useTheme();
+
+  const colorObjects = [success, primary, secondary, error, warning, info];
+
+  return [...colorObjects.map((c) => c.light), ...colorObjects.map((c) => c.main), ...colorObjects.map((c) => c.dark)];
```

Comment:
Just to be clear, the order of this doesn't really matter?  This is just a way of getting some colors that look nice and contrast in an order that ensures contrasting ones are closer together?

Reply:
The order does matter, Tiffany wants to prioritize light then main and finally dark.

Reply:
Ok yes, that is what I meant.  perfect.

Reply:
Maybe a comment makes sense here?

Reply:
That could work.  At first glance, it does seem a bit random